src/main/resources/application-dev.yml

```
@@ -1,11 +1,16 @@
 spring:
   profiles:
     include:
-      - h2
+      - mysql
```

I gues this was a mistake?
```suggestion
      - h2
```